### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by MarkVed17**
- Added Sign-in screen
- Added Sign-up screen
- Refactored Navbar component wrt auth

### CODE REVIEW

File: src/frontend/screens/SignInScreen/SignInScreen.js [hunk 1]
```diff
@@ -1,13 +1,112 @@
...
+          type="text"
+          placeholder="xyz@example.com"
+          name="email"
+          className="input standard large"
+          required
+          value={user.email}
+          onChange={(e) => setUser({ ...user, email: e.target.value })}
+        />
+
+        <label htmlFor="psw">Password</label>
```
Comment (kiranpratyush) [inline src/frontend/screens/SignInScreen/SignInScreen.js:59]: suggestion: We can give better name for HTML for

**Review by vijaytembugade — APPROVED** [review]
Code looks clean . Authentication part is also nicely declared. Good that you used a constants.

**Review by kiranpratyush — APPROVED** [review]
UI looks nice. 
can go ahead to merge

**Review by bipin7yadav — APPROVED** [review]
good to go

**Pull request merged**